packages/gamut-labs/src/EmptySection/index.tsx

```diff
@@ -43,15 +42,16 @@ const EmptyContainer = styled(FlexBox)(
     base: {
       justifySelf: 'center',
       position: 'relative',
-      px: [16, 48, 96],
-      py: [32, 48, 96],
+      px: { _: 32, sm: 64, lg: 96 },
+      py: { _: 64, lg: 96 },
+      width: 1,
     },
     variants: {
       left: {
-        pl: [0, 0, 0],
+        pl: { sm: 0 },
```

Comment:
I think I missed this when we paired:  We need to match the overrides above to guarantee ordering:
https://github.com/Codecademy/client-modules/pull/1787/files#diff-2aa502cf3c2e4c2a85b97f6dd1e970c4f2a13020bb0df60b1c88c81417ac2c66R45

It might just be easier to do:
```suggestion
        pl: { sm: 0 },
        pr: {  sm: 64, lg: 96 },
```

and `px: 32` on base and this will be easier to make sense of :)